nova/policy.py

```diff
@@ -235,6 +235,22 @@ def __call__(self, target, creds, enforcer):
         return creds['is_admin'] == self.expected
 
 
+@policy.register('target_has_no_project_id')
```

Comment:
how do you feel about a `cc_` prefix to denote that it's a ccloud-custom filter? i always liked this "namespacing" part when reading other people's code/api/config because i can quickly see what's builtin and what's custom stuff on-top.
all this under the assumption that we're not going to upstream this in any case.

i'm also fine with it as-is.

Reply:
not sure if `cc_` is enough to be recognized as our stuff by anybody reading Nova's policy.yaml. if we start prefixing, maybe `ccloud_` is better?

Reply:
fine by me. might read as "(ccloud target) has no (project id)", how about `ccloud_no_target_project_id` or `ccloud_no_project_id_in_target`?

Reply:
sure.